### DISCUSSION TITLE
Root really?

### DISCUSSION THREAD

**Discussion by Stuikerdto** [d4rken-org/sdmaid-se#223] [Q&A] [answered]
What's up with all the "need root acces" is this temporary or permanent? Didn't need this in sd maid, and I'm a big fan, because of the non root approach... So to support I bought the pro, but it doesn't do or delete anything what my other apps do automatically and all the stuff they can't or don't do is only for root... A little heads up would be nice, cause now it seems useless to me... And I don't root devices and shouldn't have to, maybe if warranty runs out, but guess what, then most of the time they don't do any security updates as well... So new phone most of the time, like oke, cool to root a 200 euro or dollar phone, if bricked meh, but not a 599 phone with good to even great customer service... So I know its early stages, but not oke to me paying for root only stuff that I can't use

**Comment by d4rken** [comment, marked as answer]
Can you elaborate, maybe some screenshots?

There was no change between SD Maid 1 and 2.
All features that required root in SD Maid 1, also require root in SD Maid 2.
Al that worked without root in SD Maid 1, also work without root in SD Maid 2.

Labeling may have changed, but not available functionality.

But maybe I made a mistake somewhere?

**Reply by moongazer07** [reply]
if shizuku support was added
less problems like this can be avoided

**Reply by Stuikerdto** [reply]
In sd maid 1 the root options were greyed out, so I didn't pay much attention to it.... So stupid of me to post this post...

Anyway glad to see sd maid back in the play store, must be a big win in the fight of google trying to control everything and people will go over to Harmony I think, I would, my phone isn't operating like it should after update 13.1, 60 apps get openen after reboot..  Put it in repair shop, upgraded software... They made it worse, I think its because of setting all google options off...
